lib/susedistribution.pm

```diff
@@ -665,7 +665,8 @@ sub hyperv_console_switch {
     # If we are in VT, 'Alt-Fx' switch already worked
     return if check_screen('any-console', 10);
     # We are in X11 and wan't to switch to VT
-    testapi::x11_start_program('xterm');
+    wait_still_screen { type_string "xinit /usr/bin/xterm -- :1 &\n" };
```

Comment:
This wait_still_screen does not assert specific screen. So it this command fails or goes sideways but 'still', then this command can still pass. You should introduce something to assert successful result.

Reply:
Will use the method assert_still_screen instead of wait_still_screen.